src/main/include/hardware/Sensors.h

```diff
@@ -14,6 +14,9 @@
 #define RIGHT_FRONT_STEER_ENCODER 2
 #define RIGHT_BACK_STEER_ENCODER 3
 
+#define EXTEND_LIMIT_SWITCH 9
+#define ARM_ABS_ENCODER 1
```

Comment:
Is this a digital input? maybe it would make sense to put a comment to separate it from being confused as colliding with the LEFT_BACK_STEER_ENCODER